R/generics.R

```diff
@@ -285,6 +285,21 @@ setMethod("obj_format<-", "ANY", function(obj, value) {
   obj
 })
 
+#' @rdname lab_name
```

Comment:
Hi @gmbecker , it took me a while to guess what "lab_name" means, I guess it is 'label_name'? I was wondering if this what you meant? if it is, we can spin name another issue and replace it to ‘label_name'?

Reply:
this only controls the physical name of the Rd file within the directory structure, so it really doesn't matter what it is (and thus doesn't seem worth changing)

Reply:
hi Gabe, this is just a minor comment, definitely not blocking this PR. I was just thinking if there are ways that we could improve, and make the code easier to read. :)